connection.go

```diff
@@ -472,28 +402,9 @@ func (conn *Connection) CloseGraceful() error {
 }
 
 // Addr returns a configured address of Tarantool socket.
-func (conn *Connection) Addr() string {
-	return conn.addr
-}
-
-// RemoteAddr returns an address of Tarantool socket.
-func (conn *Connection) RemoteAddr() string {
-	conn.mutex.Lock()
-	defer conn.mutex.Unlock()
-	if conn.c == nil {
-		return ""
-	}
-	return conn.c.RemoteAddr().String()
-}
-
-// LocalAddr returns an address of outgoing socket.
-func (conn *Connection) LocalAddr() string {
-	conn.mutex.Lock()
-	defer conn.mutex.Unlock()
-	if conn.c == nil {
-		return ""
-	}
-	return conn.c.LocalAddr().String()
+// It panics, if the connection has not been successfully established.
+func (conn *Connection) Addr() net.Addr {
+	return conn.c.Addr()
```

Comment:
```suggestion
func (conn *Connection) Addr() net.Addr {
	if conn.c != nil {
		return conn.c.Addr()
	}
	return nil
```

Reply:
When we call `reconnectImpl`, it's [calls](https://github.com/tarantool/go-tarantool/blob/39ec344a4f57d378f96738513950e660b7133fb4/connection.go#L741) `closeConnection`, which in turn [sets](https://github.com/tarantool/go-tarantool/blob/master/connection.go#L673-L676) `conn.c` to `nil` before the next success connection.